fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -969,19 +973,24 @@ private void getDeleteBitmapUpdateLock(long transactionId, List<OlapTable> mowTa
             List<Long> respBaseCompactionCnts = response.getBaseCompactionCntsList();
             List<Long> respCumulativeCompactionCnts = response.getCumulativeCompactionCntsList();
             List<Long> respCumulativePoints = response.getCumulativePointsList();
+            List<Long> respTabletStates = response.getTabletStatesList();
```

Comment:
Will there be any issues during the server version upgrade or downgrade process here?